training_config/tango/piqa.jsonnet

```diff
@@ -0,0 +1,61 @@
+local transformer_model = "bert-base-cased";
+
+local debug = true;
+
+{
+    "steps": {
+        "dataset": {
+            "type": "piqa_instances",
+            "tokenizer_name": transformer_model
+        },
+        "trained_model": {
+            "type": "training",
+            "dataset": {"ref": "dataset"},
+            "training_split": "train",
+            "data_loader": {
+                "type": "max_batches",
+                [if debug then "max_batches_per_epoch"]: 7,
```

Comment:
I think this breaks when `debug` is `false` since `max_batches_per_epoch` is a required argument for this data loader.

Reply:
You are 100% correct.